mslice/plotting/plot_window/cut_plot.py

```diff
@@ -217,6 +217,16 @@ def set_line_options_by_index(self, line_index, line_options):
 
         self._lines_visible[line_index] = line_options['shown']
 
+    def remove_line_by_index(self, line_index):
+        container = self._canvas.figure.gca().containers[line_index]
+        containers = self._canvas.figure.gca().containers
+        container[0].remove()
+        for line in container[1]:
+            line.remove()
+        for line in container[2]:
+            line.remove()
```

Comment:
Consider
`for i in range(2)`
      `for line in container[i+1]`
             `line.remove()`